docs/DeveloperGuide.md

```diff
@@ -5,25 +5,106 @@
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
 
 ## Design & implementation
-
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+### Overview
+![Overview.png](Overview.png)
+Given above is a overview of how each classes interact with one another in our software.
+
+### Save feature
+
+The save feature is facilitated by `FileSave`. It makes calls to the subclasses `Accommodation`, `Food` and `Landmark`
+which are subclasses of the `TravelActivity` class to add each activity saved in the text file.
+
+`addTravelActivity` is called everytime there is a valid-stored data in the text file that is read to upload the
+activity back into the array list.
+
+The Sequence Diagram below shows how the save file feature is being implemented when the user re-enters the chatbot
+after using the bye command.
+
+![img.png](img.png)
```

Comment:
According to the module website, class names should be on the top of the diagram, not on the bottom